src/solc_0.8/common/Libraries/SafeMathWithRequire.sol

```diff
@@ -0,0 +1,81 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.2;
+
+import "@openzeppelin/contracts-0.8/utils/math/SafeMath.sol";
+
+/**
+ * @title SafeMath
+ * @dev Math operations with safety checks that revert
+ */
+library SafeMathWithRequire {
```

Comment:
I think we don't need to use SafeMath from version 0.8.
https://docs.soliditylang.org/en/v0.8.0/080-breaking-changes.html

Reply:
Yeah I saw that. But as long as openzeplin support safemath we don't remove the safe operator to keep changes as minimal as possible.

Reply:
yes it is optional, I know.
but you know + is better than add

Reply:
Yeah I agree! For new dev we must use + - etc but when porting old source we should keep safeMath.

Reply:
it's fine, we'll use "+" for the next devs